rtl/gen_plic_addrmap.py

```diff
@@ -61,7 +61,10 @@ def addEntry(self, addr, name, description, access, width):
   """Dump Verilog"""
   def emit_verilog(self):
     output = "// Do not edit - auto-generated\n"
-    output += "module {} (\n".format(self.name)
+    params =",\n".join(map(lambda x: "  " + x, [
+      "parameter type reg_req_t  = logic",
+      "parameter type reg_rsp_t  = logic"]))
+    output += "module {} {}(\n".format(self.name, f'#(\n{params}\n)')
```

Comment:
f-strings are not available for older python versions, and I don't think they are necessary here. Is there a reason not to write this like the rest? (i.e. by adding to the `output` string)

Reply:
There is no specific reason to use them other than they look most intuitive (my personal opinion). 
Concerning not simply using `output +=`: It is just a lot neater to add more parameters that way, however, I can change that if you want me to.